### PR TITLE
Bug 1860858: About modal for regular user shouldn't error out

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @rhamilto

### CODE REVIEW

File: frontend/public/components/about-modal.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = ({ closeAboutModal, cv }
     group: ClusterVersionModel.apiGroup,
     resource: ClusterVersionModel.plural,
     verb: 'patch',
-    name: clusterVersion.metadata.name,
+    name: clusterVersion?.metadata?.name,
```
Comment (spadgett) [inline frontend/public/components/about-modal.tsx:44]: Let's hard-code `'version`'. That way the access review is correct even if `clusterVersion` resource isn't defined. The resource name is an API, so it won't change.

```suggestion
    name: 'version',
```

**Comment by spadgett** [comment]
/hold
since I had a small comment

File: frontend/public/components/about-modal.tsx (hunk 1 above)
Comment (jhadvig) [reply frontend/public/components/about-modal.tsx:44]: @spadgett maybe ?
```suggestion
    name: clusterVersion?.metadata?.name || 'version',
```

**Pull request merged**